### PR TITLE
AndroidX fixes

### PR DESCRIPTION
**Pull request by JosephSemrai**
Allows the project to build with some of the conflicts that `react-native-fbsdk` brings about (temporary solution until the dependency is updated).

Updated build process to use Jettifier and target AndroidX. Also updates annotation calls to use the new AndroidX package.

### CODE REVIEW

**Comment by viradhanus** [comment]
Still getting errors #53

**Comment by JosephSemrai** [comment]
@viradhanus Try cleaning using `gradle clean` and see if you can run it then.

Also, what's your environment info (`react-native info`) + Java version?

**Comment by viradhanus** [comment]
> @viradhanus Try cleaning using `gradle clean` and see if you can run it then.
> 
> Also, what's your environment info (`react-native info`) + Java version?
Screenshot (72)

**Comment by viradhanus** [comment]
> @viradhanus Try cleaning using `gradle clean` and see if you can run it then.
> 
> Also, what's your environment info (`react-native info`) + Java version?

I tried with ubuntu as well.. 

Screenshot from 2020-01-27 15-04-10

**Comment by viradhanus** [comment]
> > @viradhanus Try cleaning using `gradle clean` and see if you can run it then.
> > Also, what's your environment info (`react-native info`) + Java version?
> 
> I tried with ubuntu as well..
> 
> Screenshot from 2020-01-27 15-04-10

Here are the environment info

Screenshot from 2020-01-27 15-18-44

**Comment by JosephSemrai** [comment]
Hmm, I will check the issue out when I get home. There might be another
loose dependency.

**Comment by viradhanus** [comment]
Hmm, I will check the issue out when I get home. There might be another
loose dependency.

Sure keep it up... :blush:

**Comment by viradhanus** [comment]
> Hmm, I will check the issue out when I get home. There might be another loose dependency.

Sure keep it up... :blush:

**Comment by viradhanus** [comment]
@JosephSemrai 
**Can I know your environment info?**
Sometimes there may be changes that you have done, not listed here. Can you compare package.json as well?

File: android/app/src/main/java/com/trackpal/MainActivity.java [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 import com.facebook.react.ReactActivity;
 import android.content.Intent;
-import com.facebook.reactnative.androidsdk.FBSDKPackage;
 import com.facebook.react.ReactActivityDelegate;
```
[the remark this answers is not in the document]
Comment (JosephSemrai) [reply android/app/src/main/java/com/trackpal/MainActivity.java]: That might have been a remnant of me testing without the need for Facebook features. If I'm able to build the project again, I'll check out the pull request and update it.

**Comment by shehand** [comment]
Feature merged with the PR #58

**Pull request closed without merging**